server/util.go

```diff
@@ -181,23 +181,20 @@ func CreateSDKLogger(ctx *Context, out io.Writer) (log.Logger, error) {
 	}
 
 	logLvl, err := zerolog.ParseLevel(logLvlStr)
-	if err != nil {
-		// If the log level is not a valid zerolog level, then we try to parse it as a key filter.
```

Comment:
Please leave the comment.

Reply:
K, I'm not taking those out I'll treat them as linter bugs.